frontend/src/views/home.rs

```diff
@@ -0,0 +1,52 @@
+use gloo_net::http::{Headers, Request};
+use serde::{Deserialize, Serialize};
+// use serde_json;
+// use wasm_bindgen::JsValue;
+// use web_sys::HtmlInputElement;
+
+use yew::prelude::*;
+
+use crate::components::input::InputField;
+
+#[derive(Clone, PartialEq, Properties, Debug, Default, Serialize, Deserialize)]
+pub struct RegistrationForm {
```

Comment:
These types of structs are what I'd imagine would go in "common"

Reply:
We can add database migrations to set up whatever type of data we want in here. I think we should shoot for something a little more fun but this is a starter.